tests/unit-tests/shell/test_decoration_basic_decoration.cpp

```diff
@@ -133,8 +135,9 @@ struct MockShell
         uint64_t,
         MirResizeEdge));
 
-    MOCK_METHOD3(create_surface, std::shared_ptr<ms::Surface>(
+    MOCK_METHOD4(create_surface, std::shared_ptr<ms::Surface>(
```

Comment:
Please, let's use `MOCK_METHOD` for new code.

Reply:
Oh, cool, I did not know about this.